src/helpers/getCoursesDuration.js

```diff
@@ -0,0 +1,21 @@
+const getCoursesDuration = (duration) => {
+  const hours = Math.floor(duration / 60);
+  const minutes = duration % 60;
+  let durationResult = '00:00 hour';
+
+  if (duration === undefined) return durationResult;
```

Comment:
Don't forget to wrap block statements with curly brackets (use [this](https://eslint.org/docs/latest/rules/curly#rule-details) rule for updating your `.eslintrc.json`